Help removal (#2148)
openmicroscopy/ome-documentation: omero/sysadmins/cli/light-admins.rst

of the server's underlying permissions restrictions is described in
:doc:`developer documentation</developers/Server/LightAdmins>`.
OMERO.web offers easy management of restrictions and is recommended for

> setting up restricted administrators via its :guide:`Admin tab

Reviewer: ```suggestion
setting up restricted administrators as described in :guide:`Administrate Users OMERO Guide
```

> <example_facility_manager.html>`.

Reviewer: ```suggestion
<ntroduction/docs/group-user-management.html>`.
```

> <ntroduction/docs/group-user-management.html>`.

Reviewer 2: ```suggestion
<introduction/docs/group-user-management.html>`.
```
